src/main/java/net/conjur/api/Endpoints.java

```diff
@@ -32,7 +32,8 @@ public URI getSecretsUri() {
     public static Endpoints fromSystemProperties(){
 
         String account = System.getProperty(ACCOUNT_PROPERTY_NAME);
-
+        if (account == null)
```

Comment:
I know it's super small and insignificant, but as a best practice, I always suggest opening a block with `{` and `}` because this can lead to future human mistakes.

Reply:
done